operations/run-aggoracle-committee.md

```diff
@@ -0,0 +1,218 @@
+
+### Mode 2: AggOracle Committee (Decentralized)
+
+**When to use**: You want enhanced security through decentralized consensus for GER updates.
+
+**Components to run**:
+```bash
+aggkit run --cfg=/etc/aggkit/config.toml --components=aggoracle
+```
+
+**Key characteristics**:
+- Multiple oracle members participate in committee
+- Requires quorum consensus before GER injection
+- Enhanced security through decentralization
+- Committee members propose GERs, which are automatically injected when quorum is reached
+- Suitable for production environments with multiple trusted operators
+
+## Configuration Requirements
+
+### Core Configuration Sections
+
+All `aggoracle` deployments require configuration of the oracle-specific section. The configuration follows the standard AggKit format and integrates with the global configuration variables.
+
+<details>
+<summary>AggOracle configuration section</summary>
+
+```toml
+# Global configuration variables (used throughout config)
+L1URL = "https://eth-mainnet.g.alchemy.com/v2/YOUR-API-KEY"
+L2URL = "https://your-l2-rpc-endpoint"
+
+# L2 contract addresses
+[L2Config]
+    # Address of the sovereign global exit root proxy contract on L2
+    GlobalExitRootAddr = "0x0000000000000000000000000000000000000000"
+    # Address of the aggoracle committee (required for committee mode)
+    AggOracleCommitteeAddr = "0x0000000000000000000000000000000000000000"
+
+# AggOracle configuration
+[AggOracle]
+    # Enable AggOracle Committee mode (false for direct injection)
+    EnableAggOracleCommittee = false
+
+    [AggOracle.EVMSender]
+        # L2 Global Exit Root Manager contract address (references global config)
+        GlobalExitRootL2 = "{{L2Config.GlobalExitRootAddr}}"
+
+        # AggOracle Committee contract address (references global config)
+        AggOracleCommitteeAddr = "{{L2Config.AggOracleCommitteeAddr}}"
+
+        [AggOracle.EVMSender.EthTxManager]
+            # Oracle private key configuration
+            PrivateKeys = [
+                {Method = "local", Path = "/app/keystore/aggoracle.keystore", Password = "testonly"}
+            ]
+
+            [AggOracle.EVMSender.EthTxManager.Etherman]
+                # L1 Chain ID
+                L1ChainID = 1
+```
+
+</details>
+
+### Mode-Specific Configuration
+
+#### Direct Injection Mode
+
+For direct injection mode, set `EnableAggOracleCommittee = false` and ensure:
+
+1. The oracle's private key corresponds to an address that is set as the `globalExitRootUpdater` in the L2 Global Exit Root Manager contract
+2. The `AggOracleCommitteeAddr` is not required in the config file for direct injection mode
+
+#### AggOracle Committee Mode
+
+For committee mode, set `EnableAggOracleCommittee = true` and ensure:
+
+1. The oracle's private key corresponds to an address that is a member of the AggOracle Committee
+2. The `AggOracleCommitteeAddr` is set to the deployed AggOracle Committee contract address
```

Comment:
Might be good to detail a bit more where this AggOracleCommittee contract is coming from (reading the runbook from the top it feels it's suddenly introduced here without context)